src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+
+<figure class="w-figure">
+  <img src="./image17.png" alt="iOS media playback widget on the lock screen showing a podcast episode with rich metadata." width="300">
+  <figcaption class="w-figcaption">Media playing in the app can be controlled from the lock screen.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/media-session/">Media Session API</a> lets you specify metadata like artwork, track titles, etc.
+  that then gets displayed on the lock screen, smartwatches, or other media widgets in the browser.
+{% endDetails %}
+
+## Push notifications
+
+Push notifications have become a bit of an annoyance on the web
+(albeit [notification prompts are a lot quieter](https://blog.chromium.org/2020/01/introducing-quieter-permission-ui-for.html) now),
+but if used properly, they can add a lot of value.
+For example, the iOS Podcasts app can optionally notify me of new episodes of podcasts I am subscribed to or recommend me new ones, as well as alert me of new app features.
+
+<figure class="w-figure">
+  <img src="./image12.png" alt="iOS Podcasts app in the 'Notifications' settings screen showing the 'New Episodes' notifications toggle activated." width="300">
+  <figcaption class="w-figcaption">Apps can send push notifications to inform the user about new content.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://developers.google.com/web/fundamentals/push-notifications">Push API</a>
+  allows your app to receive push notifications so you can notify your users about noteworthy events around your PWA.
+  For notifications that should fire at a known time in the future and that do not require a network connection,
+  you can use the <a href="https://web.dev/notification-triggers/">Notification Triggers API</a>.
+{% endDetails %}
+
+## App icon badging
+
+Whenever there are new episodes available for one of the podcasts I am subscribed to, an app icon badge on the Podcasts home screen icon appears, again encouraging me to re-engage with the app in a way that is not intrusive.
+
+<figure class="w-figure">
+  <img src="./image13.png" alt="iOS settings screen showing the 'Badges' toggle activated." width="310">
+  <figcaption class="w-figcaption">Badges are a subtle way for applications to inform about new content.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  You can set app icon badges with the <a href="https://web.dev/badging-api/">Badging API</a>.
+  This is especially useful when your PWA has some notion of "unread" items or when you need a means
+  to unobtrusively draw the user's attention back to the app.
+{% endDetails %}
+
+## Media playback trumps energy saver settings
```

Comment:
```suggestion
## Media playback takes precedence over energy saver settings
```
I know this is overly cautious and politically correct but we should avoid the word `trump` unless absolutely needed, in this case I don't think it is. Using the word is not worth the potential backlash. High risk, low benefit.

Reply:
LOL, yeah!